scifight_proj/urls.py

```diff
@@ -13,9 +13,13 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
+from scifight import urls as scifight_urls
+from django.conf.urls import url, include
 from django.contrib import admin
+from scifight.views import index
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
-]
+    url(r'^$',              index, name='index'),
+    url(r'^admin/',         admin.site.urls),
+    url(r'^tournament/',    include(scifight_urls,)),
```

Comment:
No need for comma after `scifight_urls`.

And I'd like not to introduce the `tournament/` URL component:
```python
    url(r'^$',        index, name='index'),
    url(r'^admin/',   admin.site.urls),
    url(r'^',         include(scifight_urls)),
```